open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/server/OpenLineageRestServices.java

```diff
@@ -63,16 +63,21 @@ public LineageResponse lineage(String serverName, String userId, Scope scope, St
                     serverName,
                     methodName);
             response = openLineageHandler.lineage(scope, guid, displayNameMustContain, includeProcesses);
-        } catch (InvalidParameterException error) {
-            openLineageExceptionHandler.captureInvalidParameterException(response, error);
-        } catch (org.odpi.openmetadata.frameworks.connectors.ffdc.PropertyServerException error) {
-            openLineageExceptionHandler.capturePropertyServerException(response, error);
-        } catch (UserNotAuthorizedException error) {
-            openLineageExceptionHandler.captureUserNotAuthorizedException(response, error);
-        } catch (OpenLineageException error) {
-            openLineageExceptionHandler.captureOpenLineageException(response, error);
-        }catch (Throwable error) {
-            openLineageExceptionHandler.captureThrowable(response, error, methodName);
+        } catch (InvalidParameterException e) {
+            openLineageExceptionHandler.captureInvalidParameterException(response, e);
+            log.debug("Open Lineage Services: An exception occurred during a lineage HTTP request", e);
+        } catch (org.odpi.openmetadata.frameworks.connectors.ffdc.PropertyServerException e) {
+            openLineageExceptionHandler.capturePropertyServerException(response, e);
+            log.debug("Open Lineage Services: An exception occurred during a lineage HTTP request", e);
+        } catch (UserNotAuthorizedException e) {
+            openLineageExceptionHandler.captureUserNotAuthorizedException(response, e);
+            log.debug("Open Lineage Services: An exception occurred during a lineage HTTP request", e);
+        } catch (OpenLineageException e) {
+            openLineageExceptionHandler.captureOpenLineageException(response, e);
+            log.debug("Open Lineage Services: An exception occurred during a lineage HTTP request", e);
+        }catch (Throwable e) {
+            openLineageExceptionHandler.captureThrowable(response, e, methodName);
+            log.debug("Open Lineage Services: An exception occurred during a lineage HTTP request", e);
         }
```

Comment:
log.debug because a high number of requests are expected and these exceptions can thus occur frequently